service/surf/apps/materials/tests/tests_material_meta_tags.py

```diff
@@ -0,0 +1,61 @@
+from django.test import SimpleTestCase, Client
+from unittest.mock import patch
+
+import surf.apps.materials.views as views
+
+
+def patched_version_with_material(a, b):
+    return [{
+        'title': 'foo',
+        'description': 'bar'
+    }]
+
+
+def patched_version_without_material(a, b):
+    return []
+
+
+@patch('webpack_loader.loader.WebpackLoader.get_bundle')
```

Comment:
What does this do? I know patch, but why do we patch that 3rd party library?

Reply:
It patches webpack. Otherwise the HTML couldn't be generated because the `webpack-stats.json` was missing 😢

Reply:
Want je wilt er geen e2e test van maken? Aanvankelijk runde we prepare_e2e_tests voor alle tests en nu alleen voor e2e_tests. Dat lost wel het probleem op wat die file betreft.

Reply:
Nope, is een beetje overkill. Het is wel fijn om deze file kunnen draaien zonder eerst webpack te moeten runnen. Het is toch verder irrelevant voor deze test.